bin/tictactoe

```diff
@@ -1,3 +1,50 @@
 #!/usr/bin/env ruby
```

Comment:
Great call, I think it looks way more elegant your way.  Also, its about making everything an object so we should have done that in the first place.